example/image-classification/train_imagenet.py

```diff
@@ -52,7 +52,11 @@
 
 # network
 import importlib
-net = importlib.import_module('symbol_' + args.network).get_symbol(args.num_classes)
+if args.network == 'inception-resnet-v2':
```

Comment:
@piiswrong Is there a way to get the data shape from the dataiter  in the `symbol_inception-resnet-v2.py`? If yes I can change the code.